src/admin/index.js

```diff
@@ -522,6 +541,7 @@ module.exports = ({
     getTopicMetadata,
     events,
     fetchOffsets,
+    fetchTopicOffsets,
```

Comment:
It doesn't matter much, but let's move this so that it's under `getTopicMetadata`, just so that they're grouped somewhat logically.